doc/rule-descriptions.md

```diff
@@ -75,7 +75,7 @@ Rules that do not necessarily conform to WCAG success criterion but are industry
 | :----------------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------- | :----------------- | :----------------------------------------- | :------------------------- |
 | [accesskeys](https://dequeuniversity.com/rules/axe/3.5/accesskeys?application=RuleDescription)                                                   | Ensures every accesskey attribute value is unique                                                    | Serious            | best-practice, cat.keyboard                | failure                    |
 | [aria-allowed-role](https://dequeuniversity.com/rules/axe/3.5/aria-allowed-role?application=RuleDescription)                                     | Ensures role attribute has an appropriate value for the element                                      | Minor              | cat.aria, best-practice                    | failure, needs&nbsp;review |
-| [empty-heading](https://dequeuniversity.com/rules/axe/3.5/empty-heading?application=RuleDescription)                                             | Ensures headings have discernible text                                                               | Minor              | cat.name-role-value, best-practice         | failure                    |
+| [empty-heading](https://dequeuniversity.com/rules/axe/3.5/empty-heading?application=RuleDescription)                                             | Ensures headings have discernible text                                                               | Serious            | cat.name-role-value, best-practice         | failure                    |
```

Comment:
This should not be reported as "serious". Not sure what we can do about that without introducing a rule level impact though.